README.md

```diff
@@ -98,44 +98,95 @@ Usage: gpgbridge.rb [options]
 
 ## Example bash helper functions
 
-I have the following in my `~/.bash_profile` in WSL. Note that there is a
-line in start_gpgbridge that must be uncommented for WSL2.
+Copy the script below to somewhere accessible from WSL, make it 
+executable (`chmod +x path/to/gpgbridge_helper.sh`).
+
+Then add `source path/to/gpgbridge_helper.sh` to your `~/.bash_profile`, `~/.bashrc` or `~/.zshrc`.
+
+Make sure to edit the variables in the start of the script according to your
+setup.
 
 ```bash
+#!/usr/bin/env bash
 #--------------------------------------------------------------------------
 # GPG bridging from WSL gpg to gpg4win gpg-agent.exe
 # (needed to use a Yubikey, since WSL cannot access USB devices)
 
-if [[ -f /mnt/c/Program1/gpgbridge.rb ]]
+# Set this to true if running WSL1
+WSL2=false
```

Comment:
Maybe `WSL2=${WSL2:-false}` so that WSL2 can be set before sourcing the helper script?